docs/Containerfile.5.md

```diff
@@ -114,7 +114,7 @@ Current supported mount TYPES are bind, cache, secret and tmpfs.
 
        Common Options:
 
-              · src, source: mount source spec for bind and volume. Mandatory for bind.
+              · src, source: mount source spec for bind and volume. Mandatory for bind. If `from` is specified, `src` is the subpath in the `from` field.
 
```

Comment:
My reading is that tmpfs and secret also accept `from=`.

Reply:
Hi @dilyanpalauzov. I think `secret` only mounts `key/values` while `from` is applicable for mounts where we need `stage` or `image` so might not be applicable for secret. For `tmpfs` it might be but i need to check why buildkit does not supports this for tmpfs.

Thanks for raising this though.